### ISSUE TITLE
CheckboxInput should be grouped for screen readers

### ISSUE THREAD

**Issue by MelissaAutumn** [thunderbird/services-ui#71]
Currently if you're using screen reader it focuses on both the checkbox and the label, basically mentioning the same thing twice.

I'm not 100% sure but I think has to do with the spans in the label element:
https://github.com/thunderbird/services-ui/blob/main/src/elements/CheckboxInput.vue#L97

We could move those to `<template>` that should work since they aren't rendered and mostly use to hold v-ifs. (https://vuejs.org/guide/essentials/conditional.html#v-if-on-template)

I'm also not sure if this will fix the issue either, but an obvious thing to test.

**Comment by MelissaAutumn** [comment]
I cannot recreate this issue using Orca on linux.
